man/fi_domain.3.md

```diff
@@ -666,6 +667,78 @@ Applications can set the mr_cnt on input to fi_getinfo, in order to
 indicate their memory registration requirements.  Doing so may allow the
 provider to optimize any memory registration cache or lookup tables.
 
+## Traffic Class (tclass)
+
+The default traffic class associated with the domain and any endpoints
+created within the context of the domain.
+
+Applications can set the tclass in the domain, in order to indicate
+a request for a specific default traffic class. Otherwise, if the application sets
+the tclass field to FI_TC_TYPE_UNSPEC, the provider will set the
+default traffic class for the domain to be FI_TC_BEST_EFFORT.
```

Comment:
I would drop the reference to TC_BEST_EFFORT, and simply indicate that the provider will select a default traffic class.  That default may vary based on other attributes, such as the address or service port selected by the application.